src/Storage/Storage.Management/Models/PSContainer.cs

```diff
@@ -48,6 +48,7 @@ public PSContainer(StorageModels.ListContainerItem container)
             this.DefaultEncryptionScope = container.DefaultEncryptionScope;
             this.DenyEncryptionScopeOverride = container.DenyEncryptionScopeOverride;
             this.Deleted = container.Deleted;
+            this.ImmutableStorageWithVersioning = container.ImmutableStorageWithVersioning is null ? null : new PSImmutableStorageWithVersioning(container.ImmutableStorageWithVersioning);
```

Comment:
This is equivalant to
```suggestion
            this.ImmutableStorageWithVersioning = container.ImmutableStorageWithVersioning ?? new PSImmutableStorageWithVersioning(container.ImmutableStorageWithVersioning);
```

Reply:
fixed

Reply:
My bad - I can't use `??` between different types. Please revert the last commit. Sorry for the inconvenience!

Reply:
reverted